### DISCUSSION TITLE
How can I add a sticky on top?

### DISCUSSION THREAD

**Discussion by icepcp** [Knogobert/ntn-boilerplate#188] [Q&A] [answered]
Demo: https://cool-sticky.bss.design/

This is what I would like to replicate, is it possible?

**Comment by Knogobert** [comment, marked as answer]
I'd suggest learning up on some more HTML & CSS to achieve something like this. Then going into the way that Tailwind helps out in making it quicker to add different CSS styles by using classes.

But anyways, I'll help you out a little.

A quick solution for you is to add this as a start before the main navigation in the header:
```
<div role="banner" class="fixed top-0 inset-x-0 py-3 px-3 bg-gray z-50">This is a sticky bro (aka. banner)</div>
```

Then to knock down the main navigation a little bit, one way is to add a custom tailwind CSS class.
Add `mt-[54px]` to your main nav.
```
<nav aria-label="Main Menu" class="scrim-bg fixed z-40 top-0 inset-x-0 pt-3 px-3 mt-[54px]">
```

It will add `margin-top: 54px;`

I would also then increase the `padding-top: 4rem;` to `padding-top: 8rem;` or something on the css class `.wrapper` to make the page content jump down a bit as well.

Hope it helps, best of luck!

**Comment by icepcp** [comment]
Thank you very much! Love your boilerplate btw, it looks amazing ❤️
